tests/staticfiles_tests/test_storage.py

```diff
@@ -360,6 +360,14 @@ def test_post_processing_nonutf8(self):
         self.assertEqual("Post-processing 'nonutf8.css' failed!\n\n", err.getvalue())
         self.assertPostCondition()
 
+    def test_css_data_uri_with_nested_url(self):
+        relpath = self.hashed_file_path("cached/data_uri_with_nested_url.css")
+        with storage.staticfiles_storage.open(relpath) as relfile:
+            content = relfile.read()
+            # The data URI should be preserved as-is
```

Comment:
I think this comment is implied and can be removed. I don't think readers would assume the content should change.